### DISCUSSION TITLE
How to retain original scrollbar styling while keeping the scroll behaviour?

### DISCUSSION THREAD

**Discussion by azharzaman1** [idiotWu/smooth-scrollbar#495] [FAQ]
I want to retain the original scroll styling. I only wish to smooth scrolling.

**Comment by sadeghbarati** [comment]
I think the native scrollbar is not supported https://github.com/idiotWu/smooth-scrollbar/issues/133#issuecomment-357297518

use [`Scrollbar.detachStyle()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbardetachstyle) and ur own style that lookalike native scrollbar, but without arrows 😢

**Comment by sadeghbarati** [comment]
but you can implement arrow buttons with [`Scrollbar.addMomentum(x, y)`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbaraddmomentum)

1. append two button `up` & `down` in `scrollbar-track scrollbar-track-y` element
2. add `mousedown` events
3. on `mousedown` run 

```js
Scrollbar.addMomentum(0, -40) //up
Scrollbar.addMomentum(0, 40) //down
```

4. for simulate `mousedown + hold` you can use [requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)

> **Note**: you may need to add `debounce` or `throttle`
```js
function arrowRaf(mouseStatus, dir) {
  if(mouseStatus === 'up') {
    window.requestAnimationFrame(arrowRaf);
    return;
  }

  Scrollbar.addMomentum(0, dir === 'down' ? 40 : -40)
}
```

**Comment by sadeghbarati** [comment]
check this section of `smooth-scrollbar` source code

https://github.com/idiotWu/smooth-scrollbar/blob/25ca6b81398c01f9600bd73747f90338b6315085/src/events/keyboard.ts#L57
